Userland/Utilities/headless-browser.cpp

```diff
@@ -14,12 +14,22 @@
 #include <LibCore/ArgsParser.h>
 #include <LibCore/EventLoop.h>
 #include <LibCore/File.h>
+#include <LibCore/IODevice.h>
+#include <LibCore/MemoryStream.h>
+#include <LibCore/Stream.h>
 #include <LibCore/Timer.h>
+#include <LibGemini/GeminiRequest.h>
+#include <LibGemini/GeminiResponse.h>
+#include <LibGemini/Job.h>
```

Comment:
And then in this commit you'd also add `LibGemini LibHTTP LibWebSocket` as CMake deps